corehq/apps/cloudcare/touchforms_api.py

```diff
@@ -61,7 +61,10 @@ def get_full_context(self, root_extras=None, session_extras=None):
         # always tell touchforms to include footprinted cases in its case db
         session_data["additional_filters"] = {"footprint": True}
         session_data.update(session_extras)
-        xform_url = reverse("xform_player_proxy")
+        if toggles.EDIT_FORMPLAYER.enabled(session_data.get('domain')):
```

Comment:
although in case you do ever need to differentiate i believe you can use the device id param that gets passed in